test/verify/check-docker

```diff
@@ -25,7 +25,7 @@ from testlib import *
 @skipPackage("cockpit-docker")
 @skipImage("No docker packaged", "rhel-8-3", "rhel-8-3-distropkg", "centos-8-stream")
 @skipImage("Not supporting cockpit-docker on CoreOS", "fedora-coreos")
-@skipImage("Not supporting cockpit-docker on Debian and Ubuntu >= 20.04", "debian-stable", "debian-testing", "ubuntu-2004")
+@skipImage("Not supporting cockpit-docker on Debian and Ubuntu >= 20.04", "debian-stable", "debian-testing", "ubuntu-2004", "ubuntu-stable")
```

Comment:
I think we now only test docker on centos-7?

Reply:
Heh, indeed -- and we don't test master on centos-7 any more. I thought that we still tested it on fedora-31, as we still ship it there. We need to add a hack to the tests to boot the image with cgroupsv1, and test it there. But oh well, won't be for long any more..